less/mixins.less

```diff
@@ -0,0 +1,73 @@
+//variables
+@nav-bg : #98A4AF;
+@white-text: #FFFFFF;
+@dark-text: #222222;
+@footer-bg: #828282;
+@tab-selected: #5E9FB9;
+
+
+
+//mobile query
+@phone: (max-width: 500px);
+
+//this hides desktop images:
+.hide-img(){
+    @media @phone {
+        display:none;
+    }
+}
+
+
+
+//this displays the mobile images:
+.show-img(){
```

Comment:
Normally you don't need two separate classes for show and hide, you could just have one class, like hide, and it just overrides the properties on the base class you want to override.. 

like:
```
.image {
  width: 100%;
  display: block;

  &.hide {
    display: none;
  }
}
```